src/py-opentimelineio/opentimelineio-bindings/otio_serializableObjects.cpp

```diff
@@ -250,7 +250,13 @@ static void define_bases2(py::module m) {
         .def("next", &SerializableCollectionIterator::next);
 
     py::class_<SerializableCollection, SOWithMetadata,
-               managing_ptr<SerializableCollection>>(m, "SerializableCollection", py::dynamic_attr())
+               managing_ptr<SerializableCollection>>(m, "SerializableCollection", py::dynamic_attr(), R"docstring(
+A kind of composition which can hold any serializable object.
+
+This composition approximates the concept of a bin - a collection of :class:`.SerializableObject`\s that do
+not have any compositing meaning, but can serialize to/from OTIO correctly, with metadata and
+a named collection.
```

Comment:
A :class:`~SerializableCollection` is useful for serializing multiple timelines, clips, or media references to a single file.